packages/communication-react/CHANGELOG.md

```diff
@@ -7,114 +7,50 @@ This log was last generated on Tue, 19 Apr 2022 20:46:13 GMT and should not be m
 ## [1.2.2-beta.1](https://github.com/azure/communication-ui-library/tree/@azure/communication-react_v1.2.2-beta.1)
 
 Tue, 19 Apr 2022 20:46:13 GMT 
-[Compare changes](https://github.com/azure/communication-ui-library/compare/@azure/communication-react_v1.2.0...@azure/communication-react_v1.2.2-beta.1)
+[Compare changes](https://github.com/azure/communication-ui-library/compare/1.2.0...1.2.2-beta.1)
 
-### Minor changes
+### Features
 
-- `@internal/react-composites`
-  - Add useAzureCommunicationChatAdapter hook ([PR #1627](https://github.com/azure/communication-ui-library/pull/1627) by 82062616+prprabhu-ms@users.noreply.github.com)
-- `@internal/react-components`
-  - Introduce strings for the new modal controls. ([PR #1718](https://github.com/azure/communication-ui-library/pull/1718) by 94866715+dmceachernmsft@users.noreply.github.com)
+  - Show Error Bar to user when joining a call fails ([PR #1788](https://github.com/azure/communication-ui-library/pull/1788) by 2684369+JamesBurnside@users.noreply.github.com)
+  - Added local and remote picture-in-picture component in Chat pane of CallWithChat composite in mobile view. ([PR #1617](https://github.com/azure/communication-ui-library/pull/1617) by miguelgamis@microsoft.com)
```

Comment:
Can we make the verb tense consistent across these. Pick one of "Added" vs "Add" etc.